src/sider/historikk/Historikk.tsx

```diff
@@ -50,6 +50,7 @@ function tagFromKilde(kilde: HistorikkEventType): ReactElement {
       return <Tag variant="warning">Manglende medvirkning</Tag>;
     case "MOTEBEHOV":
     case "MOTER":
+    case "DIALOGMOTEKANDIDAT":
       return <Tag variant="warning">Dialogmøte</Tag>;
```

Comment:
Kanskje bruke noe annet enn `warning` siden vi har et par av disse allerede? 😁 Dette blir jo litt random😅

Reply:
Tenker vi burde bruke samme som `MOTEBEHOV` og `MOTER` siden alle har med dialogmøter å gjøre og disse har vi jo brukt warning på en stund.

Reply:
Tror ikke vi skal være redde for å bytte farger hvis vi ønsker det. Er bare 4% av brukerne våre som er inne på denne siden ifølge ferske amplitude-målinger, så sikkert ingen som har lagt til seg noen vaner enda 🤔

Reply:
Oki, endrer til `success` for de som har med dialogmøte å gjøre nå. Vi bør nok ta en gjennomgang av disse før vi "rebrander" siden slik at ikke blir så tilfeldig :+1: